src/client/index.js

```diff
@@ -1,3 +1,6 @@
+import "core-js/stable";
+import "regenerator-runtime/runtime";
+import "whatwg-fetch";
```

Comment:
What are we using that uses window.fetch? Loading fonts or something?

Reply:
I think it was the http backend for i18next, it loads the content / translation files